daq/device_qualification_client.py

```diff
@@ -9,7 +9,7 @@
 _SERVER_PORT_DEFAULT = 50051
 
 
-class DeviceTestingClient:
+class DeviceQualificationClient:
```

Comment:
Qualification is not right either. That's a description of the USE of it, not what it is.  Also, this entire repo is "qualification" so this name could apply to any client that's in the repo.   What is it connecting to on the other end?  B/c it's a client of that and that's what should be in the name.

Reply:
It is connecting to `DeviceTestingServer` in forch... But I think that should be renamed as well... What about `DevicePortBehaviorClient` or `DeviceResultClient`?